packages/common/src/ui/components/index.ts

```diff
@@ -1,8 +1,9 @@
+import Collapse from '@material-ui/core/Collapse';
 import Divider from '@material-ui/core/Divider';
 import Typography from '@material-ui/core/Typography';
 import CircularProgress from '@material-ui/core/CircularProgress';
 
 export * from './LoadingApp';
 export * from './RemoteComponent';
 export { AppNavLink } from './NavLink';
-export { Typography, CircularProgress, Divider };
+export { Typography, CircularProgress, Divider, Collapse };
```

Comment:
already is 😉 
Just trying to keep the bundle small. I hoped that we hit a point where we are just reusing things rather than adding.. but that might be a long way down the road?